package.json

```diff
@@ -32,18 +31,15 @@
   },
   "homepage": "https://github.com/visualjerk/quill-magic-url",
   "devDependencies": {
-    "babel-cli": "^6.26.0",
-    "babel-core": "^6.26.0",
-    "babel-preset-es2015": "^6.24.1",
-    "babel-preset-stage-1": "^6.24.1",
-    "eslint": "4.14.0",
-    "webpack": "^4.14.0",
-    "webpack-cli": "^3.0.8",
-    "webpack-dev-server": "^3.1.4"
```

Comment:
Please keep webpack-dev-server as a dev dependency, as it is used for the development script (`npm run dev`)